static/src/javascripts/projects/commercial/modules/messenger/get-stylesheet.spec.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import { _ } from './get-stylesheet';
 
-const { getStyles } = _;
+const getStyles: any = _.getStyles;
```

Comment:
Yeah this function expects a second argument of type StyleSheetList, and we want to be able to give it an array for ease of testing. I removed Kate's earlier comments:
```
 53  // We are mocking DOM stuff, so end up setting up styleSheets		
 54  // as an Array, rather than the expected StlyeSheetList... $FlowFixMe
```